utils: add run_stopped_at to workflow progress info (#516)
reanahub/reana-workflow-controller: CHANGES.rst

Version 0.9.1 (UNRELEASED)
--------------------------

> - Adds ``run_stopped_at`` to the list of properties returned by ``Workflow.get_workflow_status``.

Reviewer: ```suggestion
- Adds the timestamp of when the workflow was stopped (``run_stopped_at``) to the workflow list and the workflow status endpoints.
```

> - Adds ``run_stopped_at`` to the list of properties returned by ``Workflow.get_workflow_status``.

Reviewer: ```suggestion
- Adds the timestamp of when the workflow was stopped (``run_stopped_at``) to the workflow list and the workflow status endpoints.
```
